core/server/master/src/main/java/alluxio/master/file/meta/LockingScheme.java

```diff
@@ -75,7 +75,13 @@ public LockPattern getDesiredPattern() {
   }
 
   /**
-   * @return the mode that should be used to lock the path, considering if ufs sync should occur
+   * Gets the lock pattern for this locking scheme.
```

Comment:
I believe that `Gets the lock pattern for this locking scheme and return it` is better. What do youu think?